recipe-server/normandy/recipes/tests/api/v2/test_serializers.py

```diff
@@ -64,17 +64,27 @@ def test_it_works(self, rf):
             'identicon_seed': Whatever.startswith('v1:'),
         }
 
+    def test_validation_with_invalid_action(self):
```

Comment:
How is this test different than `test_validation_with_wrong_action`, below?

Reply:
this time the `action_id` is a string. If you let that through to `Action.objects.get(pk='some string')` you get a ORM `ValueError`.

Reply:
It's a string in both cases, just in the other test it is the string '9999'.

Reply:
The `'9999'` gets converted to an int. Check my followup commit that really checks the error messages using `default_error_messages`. One sec.

Reply:
Weird. Thanks for talking through this with me.